templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,83 @@
+# Analytics
+
+Zulip provides analytics for tracking various usage statistics about you and
+your organization.
+So, you can get a holistic view of your organization's communications.
+
+Analytics can be accessed via the [stats](/stats) page.
+
+### Messages Sent Over Time
+
+You can view number of messages that have been sent over a particular time frame.
+
+* You can change the **Aggregation** to view the number of messages sent on an
+hourly, daily, weekly basis, or the cumulative amount.
+
+    ![Aggregation](/static/images/help/stats-aggregation.png)
+
+* You can change the time frame of messages to **Last 2 months**, **Last 6 months**,
+or **All time**.
+
+    ![Sliding window](/static/images/help/stats-sliding-window.png)
+
+    A custom time frame is also possible by clicking and sliding the handle in
+    the bottom window.
+
+* By default the number of messages sent by **Humans** (users) is shown, but
+you can also view the number of messages sent by bots, by clicking on **Bots** at the
+top right.
+
+    ![Humans-Bots](/static/images/help/stats-humans-bots.png)
+
+!!! tip ""
+    To view the exact number of number of messages, just hover over the graph and
+    notice the count change on the bottom left.
+
+### Messages Sent by Client
+
+You can view the message distribution based on which client (platform) was
+used for sending them.
+
+* You can change the time frame of messages to **Last week**, **Last month**,
+**Last year**, or **All time**.
+
+* You can also change the view to messages sent by you (**Me**) or by everyone in
+your organization (**Everyone**).
+
+    ![Message-by-client](/static/images/help/stats-message-by-client.png)
+
+### Messages Sent by Recipient Type
+
+You can view the message distribution based on whom the message was intended for.
+This includes public streams, personal messages (PMs) and private streams.
+
+* You can change the time frame of messages to **Last week**, **Last month**, or
+**All time**.
+
+* You can also change the view to messages sent by you (**Me**) or by everyone in
+your organization (**Everyone**).
+
+* Total number of messages for the above selected options is shown separately at the
```

Comment:
This can be worded in simpler words. "Total number of sent messages is shown separately at the bottom left."